### PR TITLE
[Strapi] Create Test Content Type

### PR DESCRIPTION
**Pull request by tungto**
https://datisan.atlassian.net/browse/XDLD-179

### CODE REVIEW

File: apps/cms/src/api/test/content-types/test/schema.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+      "unique": true
+    },
+    "passingScore": {
+      "type": "decimal",
+      "required": true,
+      "min": 0,
+      "max": 100,
+      "default": 0
+    },
+    "status": {
```
Comment (nvt1904) [inline apps/cms/src/api/test/content-types/test/schema.json:27]: We have draftAndPublish option model 
https://docs.strapi.io/dev-docs/backend-customization/models#model-options
https://docs.strapi.io/user-docs/content-manager/saving-and-publishing-content#publishing-a-draft

File: apps/cms/src/api/test/content-types/test/schema.json [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      "required": true
+    },
+    "timeLimit": {
+      "type": "time",
+      "required": true,
+      "default": "00:00"
+    },
+    "questions": {
+      "type": "dynamiczone",
+      "components": [
```
Comment (nvt1904) [inline apps/cms/src/api/test/content-types/test/schema.json:43]: Wait for this PR to be merged https://github.com/xponvn/test-assessment/pull/2. we will use question.choice-question and question.text-answer-question for components. like this
`
"components": [
        "question.choice-question",
        "question.text-answer-question"
 ]
`

File: apps/cms/src/components/question/question.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (nvt1904) [inline apps/cms/src/components/question/question.json:1]: We will use question.choice-question and question.text-answer-question, please remove it

Comment (tungto) [reply apps/cms/src/components/question/question.json:1]: updated

**Review by nvt1904 — APPROVED** [review]

**Pull request merged**